src/biomappings/resources/unsure.tsv

```diff
@@ -9,6 +9,7 @@ ccle	ECC4	ECC4	skos:exactMatch	cellosaurus	CVCL_1190	ECC4	manually_reviewed	orci
 ccle	ECC4	ECC4	skos:exactMatch	depmap	ACH-002024	ECC4	manually_reviewed	orcid:0000-0001-9439-5346
 ccle	NCIH1048_LUNG	NCI-H1048	skos:exactMatch	efo	0002248	NCI-H1048	manually_reviewed	orcid:0000-0001-9439-5346
 ccle	TE4_OESOPHAGUS	TE-4	skos:exactMatch	cellosaurus	CVCL_F784	TE4 [Mouse hybridoma]	manually_reviewed	orcid:0000-0003-4423-4370
+chebi	CHEBI:135548	dioxyline	skos:exactMatch	mesh	C004856	dimoxyline	manually_reviewed	orcid:0000-0001-9439-5346
```

Comment:
these are the same - they have matching CAS numbers.

Reply:
I see, I didn't notice that, I'll try to manually move it over.

Reply:
Also, you might have seen https://github.com/sorgerlab/indra/pull/1376 which actually tries to exploit precisely such MESH-CAS-CHEBI paths.